src/terminal/adapter/adaptDispatch.cpp

```diff
@@ -3738,6 +3738,73 @@ bool AdaptDispatch::DoFinalTermAction(const std::wstring_view string)
     // modify the state of that mark as we go.
     return false;
 }
+// Method Description:
+// - Performs a XtermJs action
+// - Ascribes to the ITermDispatch interface
+// - Currently, the actions we support are:
+//   * Completions: An experimental protocol for passing shell completion
+//     information from the shell to the terminal. This sequence is still under
+//     active development, and subject to change.
+// - Not actually used in conhost
+// Arguments:
+// - string: contains the parameters that define which action we do
+// Return Value:
+// - false in conhost, true for the SetMark action, otherwise false.
+bool AdaptDispatch::DoXtermJsAction(const std::wstring_view string)
+{
+    // This is not implemented in conhost.
+    if (_api.IsConsolePty())
+    {
+        // Flush the frame manually, to make sure marks end up on the right line, like the alt buffer sequence.
+        _renderer.TriggerFlush(false);
+        return false;
+    }
+
+    if constexpr (!Feature_ShellCompletions::IsEnabled())
```

Comment:
FYI, you said in the intro that this was only enabled on Dev builds but it is using the same feature flag as the one that's enabled on preview builds.

> It's disabled in Release & Preview builds

Reply:
Op yea, we later discussed leaving it on for preview builds (but behind the setting) so that we could actually get 3p feedback on the UX. 

This is a good candidate for moving to a nightly flag